drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModelTest.java

```diff
@@ -568,4 +569,37 @@ private void commonValidatePropertyUpdate(int columnIndex, boolean isPropertyTyp
         }
         reset(eventBusMock);
     }
+
+    @Test
+    public void resetError() {
+        scenarioGridModel = spy(new ScenarioGridModel(false) {
```

Comment:
Why don't you reuse the existing `scenarioGridModel`? If you need a different initialization please update the existing one in setup method

Reply:
Because I wanted to silence method "refreshErrors()" for this test only. If I put it in "setup()", that method will be silenced in all the tests defined in the class. For example, test of "refreshErrors()" at line 470 will fail.